diff --git a/libstdc++-v3/src/c++17/fs_ops.cc b/libstdc++-v3/src/c++17/fs_ops.cc
--- a/libstdc++-v3/src/c++17/fs_ops.cc
+++ b/libstdc++-v3/src/c++17/fs_ops.cc
@@ -1123,7 +1123,6 @@ fs::permissions(const path& p, perms prms, perm_options opts,
     }
 
   int err = 0;
-#ifndef __DEVKITPPC__
 #if _GLIBCXX_USE_FCHMODAT
   const int flag = (nofollow && is_symlink(st)) ? AT_SYMLINK_NOFOLLOW : 0;
   if (::fchmodat(AT_FDCWD, p.c_str(), static_cast<mode_t>(prms), flag))
@@ -1133,7 +1132,6 @@ fs::permissions(const path& p, perms prms, perm_options opts,
     ec = std::__unsupported();
   else if (posix::chmod(p.c_str(), static_cast<posix::mode_t>(prms)))
     err = errno;
-#endif
 #endif
 
   if (err)
diff --git a/libstdc++-v3/src/filesystem/ops.cc b/libstdc++-v3/src/filesystem/ops.cc
--- a/libstdc++-v3/src/filesystem/ops.cc
+++ b/libstdc++-v3/src/filesystem/ops.cc
@@ -972,7 +972,6 @@ fs::permissions(const path& p, perms prms, error_code& ec) noexcept
     }
 
   int err = 0;
-#ifndef __DEVKITPPC__
 #if _GLIBCXX_USE_FCHMODAT
   const int flag = (nofollow && is_symlink(st)) ? AT_SYMLINK_NOFOLLOW : 0;
   if (::fchmodat(AT_FDCWD, p.c_str(), static_cast<mode_t>(prms), flag))
@@ -982,7 +981,6 @@ fs::permissions(const path& p, perms prms, error_code& ec) noexcept
     ec = std::__unsupported();
   else if (posix::chmod(p.c_str(), static_cast<mode_t>(prms)))
     err = errno;
-#endif
 #endif
 
   if (err)
